bundles/org.openhab.binding.systeminfo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -298,6 +298,10 @@
 		<item-type>Number:Time</item-type>
 		<label>Remaining Time</label>
 		<description>Remaining time</description>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
This seems to be about (estimated) remaining time on battery. Should be `Calculation` or `Status`.
But as this seems to be the only tagging in overall binding and the channel is one of the more exotic ones, I suggest to remove it.